editor/editor_settings.h

```diff
@@ -134,6 +134,7 @@ class EditorSettings : public Resource {
 
 	static void create();
 	void setup_language();
+	String get_online_docs_url();
```

Comment:
`editor_settings.h` seems like the wrong place for this function, but nothing else seemed exactly right. I looked at `translation_server.h`, `editor_help.h`, and `editor_translation.h`.